sdks/python/apache_beam/ml/gcp/recommendations_ai_test.py

```diff
@@ -0,0 +1,208 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Unit tests for Recommendations AI transforms."""
+
+from __future__ import absolute_import
+
+import mock
+import unittest
+
+import apache_beam as beam
+from apache_beam.metrics import MetricsFilter
+
+try:
+  from google.cloud import recommendationengine
+
+  from apache_beam.ml.gcp import recommendations_ai
+except ImportError:
+  recommendationengine = None
+  raise ImportError(
+      'Google Cloud Recommendation AI not supported for this execution '
+      'environment (could not import google.cloud.recommendationengine).')
```

Comment:
Avoid raising this ImportError here - when `recommendationengine` is None, this test will not run. This should fix the import errors on the tests.